src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
> That just moves the detection from either based on a file to based on a directory name.
> You will need to recognize the possible different directory names as the detection then.

No. You simply know that if there is a directory
xx/msdos then we have msdos. If there is a directory
pcdos then we have pcdos. With --list you just do "ls"
to find which doses are there.
So the detection of msdos is:
if [ test -d "$MSDOS_DIR" ]; then
  echo we have msdos
fi